examples/CSVImporter/CSVImporter_test.go

```diff
@@ -0,0 +1,122 @@
+package main
+
+import (
+	"os"
+	"testing"
+	"github.com/TheJumpCloud/jcapi"
+)
+
+const (
+	testUrlBase string = "https://console.jumpcloud.com/api"
+	authUrlBase string = "https://auth.jumpcloud.com"
+)
+
+var testAPIKey string = os.Getenv("JUMPCLOUD_APIKEY")
+var testSystemID string = os.Getenv("JUMPCLOUD_SYSTEMID")
+
+
+func TestCSVImporter(t *testing.T) {
+	// Attach to JumpCloud
+	jc := jcapi.NewJCAPI(testAPIKey, testUrlBase)
+
+	// Fetch all users in JumpCloud
+	userList, err := jc.GetSystemUsers(false)
+
+	if err != nil {
+		t.Fatalf("Could not read system users, err='%s'\n", err)
+		return
+	}
+
+	// Fetch our system from JumpCloud
+	system, err := jc.GetSystemById(testSystemID, true)
+
+	if err != nil {
+		t.Fatalf("Could not read system info for ID='%s', err='%s'\n", testSystemID, err)
+		return
+	}
+
+	if system.Hostname == "" {
+		t.Fatalf("Could not read system info for ID='%s', err='%s'\n", testSystemID, err)
+		return
+	}
+
+	// Create a CSV record to add a test user
+	csvrec := []string{"Joe", "Smith", "js", "TheMan@jumpcloud.com", "", "", "T", "", ""}
```

Comment:
Not critical for this round, just a suggestion: instead of passing in a pre-parsed string array, it would be a more encompassing test to pass in a raw string, and let the reader parse it. That will give you more control over the various conditions you test, and allow you to make sure things don't go south with really funky input.